docs/_tutorials/reference.md

```diff
@@ -0,0 +1,138 @@
+| `client` | All listeners | Web API client that uses the token associated with that event. For single-workspace installations, the token is provided to the constructor. For multi-workspace installations, the token is returned by the `authorize` function.
+| `respond` | `shortcut`, `action`, `command` | Function that responds to an incoming event **if** it contains a `response_url`. `respond` returns a promise that resolves with the results of responding using the `response_url`.
+| `context` | All listeners | Event context. This object contains data about the event and the app, such as the `botId`. Middleware can add additional context before the event is passed to listeners.
+| `body` | All listeners | Object that contains the entire body of the request (superset of `payload`). Some accessory data is only available outside of the payload (such as `trigger_id` and `authorizations`).
+
+#### Body and payload references
+The structure of the `payload` and `body` is detailed on the API site:
+- `action`: [`body`](https://api.slack.com/reference/interaction-payloads/block-actions) and [`payload`s](https://api.slack.com/reference/block-kit/block-elements)
+- `event`: [`body`](https://api.slack.com/types/event) and [`payload`s](https://api.slack.com/events)
+- `shortcut`: [`body` and `payload`](https://api.slack.com/reference/interaction-payloads/shortcuts)
+- `command`: [`body` ](https://api.slack.com/interactivity/slash-commands)
+- `view`: [`view_submission` `body` and `payload`](https://api.slack.com/reference/interaction-payloads/views#view_submission); [`view_closed` `body` and `payload`](https://api.slack.com/reference/interaction-payloads/views#view_closed)
+- `options`: [`body`] and [`payload`](https://api.slack.com/reference/block-kit/block-elements#external_select)
+
+## Initialization options
+Bolt includes a collection of initialization options to customize apps. There are two primary kinds of options: Bolt app options and receiver options. The receiver options may change based on the receiver your app uses. The following receiver options are for the default `ExpressReceiver` (so they'll work as long as you aren't using a custom receiver).
+
+### Receiver options
+`ExpressReceiver` options can be passed into the app constructor, just like the Bolt app options.
+
+| Option  | Description  |
+| :---: | :--- |
+| `signingSecret` | A `string` from your app's configuration (under "Basic Information") which verifies that incoming events are coming from Slack |
+| `endpoints` | A `string` or `object` that specifies the endpoint(s) that the receiver will listen for incoming requests from Slack. For `object`, the `key` is the type of events (ex: `events`), and the value is the endpoint (ex: `/myapp/events`). **By default, all events are sent to the `/slack/events` endpoint** |
```

Comment:
Developers should be curious about the list of possible keys.

Reply:
Hmmm yes, I think so too. As far as I'm aware, there aren't any preset keys for different events, so should I just remove that portion?

Reply:
I think we can describe the current state - although this is a key-value object, the only available key at this time is "events"